source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -279,9 +250,9 @@ const selectedFeatureLoaderStates = {
 };
 
 const sideBarStates = {
-  initial: 'fullScreen',
+  initial: 'close',
```

Comment:
nit: given that these are states and not actions I'd rather these were 'closed' and 'opened'

Reply:
yeah, I struggled between the names. I will set it to closed'